### PR TITLE
Move to web-based ticketing system from the bubble-chat style

### PR DESCRIPTION
**Pull request by wisdeom**
Updating Readme in regards to customer support since we're moving away from the bubble-chat style of interaction to email-based interaction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ To get a top level list of the available commands, run `appcenter help`.
 
 To get help on a specific command or category, run `appcenter help command` or pass the `-h` flag to any command or category name.
 
-App Center SDK support is provided directly within the App Center portal. Any time you need help, just log in to [App Center](https://appcenter.ms), then click the blue chat button in the lower-right corner of any page and our dedicated support team will respond to your questions and feedback. For additional information, see the [App Center Help Center](https://intercom.help/appcenter/getting-started/welcome-to-app-center-support).
+App Center SDK support is provided directly within the App Center portal. Any time you need help, just log in to [App Center](https://appcenter.ms), then click on **'Contact support'** inside the "?" menu on the top right of the App Center portal and our dedicated support team will respond to your questions and feedback.
```
Comment (anywherepilot) [inline README.md:25]: is provided -> App Center provides

Comment (anywherepilot) [inline README.md:25]: log in -> sign in https://review.docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/log-on-log-off?branch=master

Comment (anywherepilot) [inline README.md:25]: click -> select/choose https://review.docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click?branch=master

File: src/commands/test/lib/error-info-builder.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export function buildErrorInfo(err: any, profile: Profile, command: AppCommand):
   const exitCode = err.exitCode || err.errorCode || ErrorCodes.Exception;
   let message : string = null;
 
-  let helpMessage = `Further error details: For help, please send both the reported error above and the following environment information to us by going to https://appcenter.ms/apps and starting a new conversation (using the icon in the bottom right corner of the screen)${os.EOL}
+  let helpMessage = `Further error details: For help, please send both the reported error above and the following environment information to us by going to https://appcenter.ms/apps and open a support ticket (using the "?" menu in the top right corner of the screen)${os.EOL}
```
Comment (anywherepilot) [inline src/commands/test/lib/error-info-builder.ts:9]: open -> opening (since it says 'going' earlier)
and top -> upper

File: README.md (hunk 1 above)
Comment (anywherepilot) [inline README.md:25]: top right -> upper right https://review.docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/upper-left-upper-right?branch=master

File: src/commands/test/lib/error-info-builder.ts (hunk 2 above)
Comment (wisdeom) [reply src/commands/test/lib/error-info-builder.ts]: @anywherepilot Thanks! Do you mind elaborating on the difference between 'and top' and 'upper'?

Comment (anywherepilot) [reply src/commands/test/lib/error-info-builder.ts]: I should use quotes ^^ I meant and "top" -> "upper". You already changed it.

File: README.md (hunk 1 above)
Comment (wisdeom) [reply README.md]: @anywherepilot Thanks!

**Comment by jp-andre** [comment]
@wisdeom there is a merge conflict on this PR. I think we already merged a similar change yesterday. Can you please resolve the conflict or close this PR? :)

**Comment by anywherepilot** [comment]
The conflict is due to prettier.

**Pull request merged**